packages/nimble-components/src/accordion/specs/README.md

```diff
@@ -0,0 +1,137 @@
+    </nimble-accordion-item>
+    <nimble-accordion-item>
+        <nimble-checkbox slot="start"></nimble-checkbox>
+        <div slot="heading">Accordion three</div>
+        <nimble-select slot="end">
+            <nimble-option value="1">Option 1</nimble-option>
+            <nimble-option value="2">Option 2</nimble-option>
+            <nimble-option value="3">Option 3</nimble-option>
+        </nimble-select>
+    </nimble-accordion-item>
+    <nimble-accordion-item>
+        <nimble-checkbox slot="start"></nimble-checkbox>
+        <div slot="heading">Accordion four</div>
+        <nimble-number-field placeholder="number" slot="end"></nimble-number-field>
+    </nimble-accordion-item>
+</nimble-accordion>
+```
+
+### API
+
+#### Nimble Accordion
+
+[FAST accordion API](https://github.com/microsoft/fast/blob/57f3c22c6341d8a21d48b1ffb7fcbfab1ffd02d8/packages/web-components/fast-foundation/src/accordion/accordion.spec.md)
+
+-   _Component Name:_ `nimble-accordion`
+-   _Properties/Attributes:_
+    -   `appearance` - `block`, `outline`, `ghost`
+-   _Methods:_ Unchanged
+-   _Events:_ Unchanged
+-   _CSS Classes and Custom Properties that affect the component:_ Unchanged
+-   _Slots:_ Unchanged
+
+The Figma design includes appearances of the accordion header that reflect those of the `block`, `outline`, and `ghost` appearances used in other components. Because of the difference in border functionality and possibility of excess overridden css classes if patterns are used, shared styles will not be used, and new styles for the accordion will be created. The appearances of `block`, `outline`, and `ghost` will be implemented under the HTML attribute `appearance`, which will use conditional css styles based on `appearance`'s value (ex. :host([appearance='outline']) will have styling for the outline button).
+
+Documentation will be added to advise against using multiple Nimble Accordions with different appearances next to each other.
+
+#### Nimble Accordion Item
+
+[FAST accordion-item API](https://github.com/microsoft/fast/tree/57f3c22c6341d8a21d48b1ffb7fcbfab1ffd02d8/packages/web-components/fast-foundation/src/accordion-item)
+
+-   _Component Name:_ `nimble-accordion-item`
+-   _Properties/Attributes:_
+    -   `error-visible` - boolean
+-   _Methods:_ Unchanged
+-   _Events:_ Unchanged
+-   _CSS Classes and Custom Properties that affect the component:_ Unchanged
+-   _Slots:_ Unchanged
+
+The Figma design also includes an `error` state. This will be controlled with the boolean attribute `error-visible`, which has a default attribute value of "". When needed, "error-visible" would be added to the accordion attributes, changing the color of the accordion item border color to red. This will be implemented in the accordion styling through conditional css styles based on `error-visible`'s value (ex. :host([error-visible]) will have styling for when `error-visible` is true).
```

Comment:
It's interesting that the design shows a way to indicate that there's an error, but no way to indicate what that error is. Assuming that's not an oversight, I wonder if the idea is that an accordion item would show an error if-and-only-if something in its content is in an error state, and that the content would be responsible for explaining what the error is.

If that's the case, we should plan to add usage guidance explaining that in the storybook documentation for `error-visible`. An alternative could be that the accordion detects that by scanning its children for error states and enters its own error state automatically, but I'd vote that's overkill. 

When he's back, can you double check with Brandon about this?

Reply:
That's what Mert and I assumed the functionality of the error state would be, and I'll add a note in the spec to make sure we create that guidance in the docs. Sure, I'll let you know what he says!